lib/OpenQA/Worker.pm

```diff
@@ -767,7 +767,8 @@ sub _check_system_utilization (
     return undef unless $threshold && @$load >= 3;
     # look at the load evolution over time to react quick enough if the load
     # rises but accept a falling edge
-    return "The average load (@$load) is exceeding the configured threshold of $threshold."
+    return
+"The average load (@$load) is exceeding the configured threshold of $threshold. The worker will temporarily not accept new jobs until the load is lower again."
```

Comment:
one comment quite off the PR. this seems to be displayed as a info next to the status. I wonder why we do not display the load as another key:value field in the page.

Reply:
not sure. Maybe check with others too. for me makes sense but maybe there is a reason

Reply:
The load is not checked every time the worker details page is shown. The worker sends this info via websocket if the load is too high, and it gets saved into the worker table in the database.